new-pipeline/src/evo.js

```diff
@@ -394,8 +397,12 @@ var gLoadedAggregatesFromState = false;
 function updateAggregates(kind, buckets) {
   gCurrentKind = kind;
   let newAggregates;
+  const IncludeSpill = $("input[name=include-spill-toggle]").is(":checked");
+
   if (kind === "enumerated" || kind === "categorical") {
     newAggregates = getHumanReadableBucketOptions(kind, buckets);
+    // Hide spill Option from Aggregates based on advanced settings
+    if(!IncludeSpill) newAggregates = newAggregates.filter(([key,value]) => value !== "spill");
```

Comment:
Conditionals should be a minimum of three lines
```patch
-if (cond) statement();
+if (cond) {
+  statement();
+}
```

(also, space required between `if` and `(`)